app/experimenter/legacy-ui/rapid/components/experiments/ResultsTable.tsx

```diff
@@ -0,0 +1,270 @@
+          <i
+            className="fas fa-arrow-down"
+            data-tip={SIGNIFICANCE_TIPS.NEGATIVE}
+          />
+        );
+        className = "negative-significance";
+        break;
+      case Significance.NEUTRAL:
+        significanceIcon = (
+          <i className="fas fa-minus" data-tip={SIGNIFICANCE_TIPS.NEUTRAL} />
+        );
+        className = "neutral-significance";
+        break;
+    }
+
+    return (
+      <div className={className}>
+        <ReactTooltip />
+        {significanceIcon}
+        &nbsp;
+        {interval}
+      </div>
+    );
+  }
+};
+
+const countField = (lower, upper, significance) => {
+  const interval = (
+    <>
+      {Math.round(lower)} to {Math.round(upper)}
+    </>
+  );
+  return showSignificanceField(significance, interval);
+};
+
+const percentField = (lower, upper, significance) => {
+  const interval = (
+    <>
+      {Math.round(lower * 1000) / 10}% to {Math.round(upper * 1000) / 10}%
+    </>
+  );
+  return showSignificanceField(significance, interval);
+};
+
+const ResultsTable: React.FC<{ experimentData: ExperimentData }> = ({
+  experimentData,
+}) => {
+  const { resultsMetricsList, resultsMetricsMap } = getResultMetrics(
+    experimentData.features,
+  );
+  const resultsData = getResultsData(experimentData, resultsMetricsMap);
+
+  return (
+    <table
+      className="table text-right analysis-table"
+      data-testid="results-table"
+    >
+      <thead>
+        <tr>
+          <th scope="col"></th>
+          {resultsMetricsList.map((value, index) => {
+            return (
+              <th key={index} scope="col">
+                {value.name}
+                &nbsp;
+                <i
+                  className="far fa-question-circle"
+                  data-tip={value.tooltip}
+                />
+              </th>
+            );
+          })}
+        </tr>
+      </thead>
+      <tbody>
+        {Object.keys(resultsData).map((branch, index) => {
+          return (
+            <tr key={index}>
+              <th className="align-middle" scope="row">
+                {branch}
+              </th>
+              {resultsMetricsList.map((metric, index) => {
+                const metricKey = metric["value"];
+                const {
+                  lower,
+                  upper,
+                  point,
+                  percent,
+                  significance,
+                } = resultsData[branch][metricKey];
+
+                let field;
+                switch (metricKey) {
+                  case "identity":
+                    field = populationField(point, percent);
+                    break;
+                  case "search_count":
+                    field = countField(lower, upper, significance);
+                    break;
+                  default:
+                    field = percentField(lower, upper, significance);
+                }
+
+                return (
+                  <td key={index} className="align-middle">
```

Comment:
For each of these mapped renders, is there another unique value that can be used instead of `index`? While it probably looks and works fine in its current state, `index` can lead to problems down the road if the array is modified. More info [here](https://medium.com/@robinpokorny/index-as-a-key-is-an-anti-pattern-e0349aece318) and hat-tip @LZoog for bringing this to my attention.